src/app/components/observation/observation/observation.component.ts

```diff
@@ -17,8 +19,9 @@ import {
   IonChip,
   IonIcon,
   IonLabel,
+  ToastController,
 } from '@ionic/angular/standalone';
-import { RegistrationViewModel } from 'src/app/modules/common-regobs-api';
+import { RegistrationService, RegistrationViewModel } from 'src/app/modules/common-regobs-api';
 import { addIcons } from 'ionicons';
 import {
   calendarNumberOutline,
```

Comment:
du trenger å importere ikonen her og så legge den til i addIcons metoden for å kunne vise den